### ISSUE TITLE
Minor Issues as of build #73

### ISSUE THREAD

**Issue by TheMasterBlaster** [DragonetMC/DragonProxy#17]
#1. Entity head rotation incorrect (Always facing south)
#2. Entitys jump 1-2 blocks in the air when moving
#3. PE sees PC client 2-3 blocks lower then it should be (if PE sees PC client at all)
#4. Block removal does not update graphically
#5. On PE crash PE client does not always disconnect.
#6. Gamemode update from Survival to Creative for PE does not take effect for PE client (even after reloging).
#7. The time of the server does not sync for PE client.
#8. PE client cant see mobs with items in their hands. (possibly even armor but have not checked yet)
#9. PE client does not see Redstone being placed but PC client until updated by a placed blocks.
#10.  PE client does not update to show when Entities are on fire.
#11. PE client sees Entities floating ~8-10 blocks higher when swimming.
#12. PE client does not receive Weather updates. 
#13. PE client does not see Item Entities. (not even after relog)
#14. PE client does not receive LightningBolt Entity update.
#15. Double Tall Grass for PE client shows at 2 lower halfs.
#16. PE clients crash on PC client join.

List will continuously be updated for each updated build and when more Minor Issues are found.

**Comment by ExtaSpace** [comment]
I want to add, that DragonProxy shows "Client disconnected" when trying to leave lobby on server. (Command: /server skyblock). But PE doesn't show, that it is disconnected.

**Comment by DefinitlyEvil** [comment]
@ExtaSpace Any errors?

**Comment by DefinitlyEvil** [comment]
As for No.5, server will automatically disconnect client for the reason of timeout. As for No.24, that was a client problem not our's.

**Comment by TheMasterBlaster** [comment]
I added Number 24 because it does not do it In Single player or when joining someone elses map

**Comment by DefinitlyEvil** [comment]
@TheMasterBlaster Hmm, I have no idea cuz I translated all blocks correctly.

**Comment by TheMasterBlaster** [comment]
It mainly does it on generated blocks but not placed blocks.

**Comment by DefinitlyEvil** [comment]
@TheMasterBlaster yes it is, I believe this is a MCPE bug.

**Comment by ExtaSpace** [comment]
@DefinitlyEvil No errors. Just says, that client disconnected. I will try the newest build, and report.

**Comment by seanth** [comment]
Can confirm #16. Running a Bukkit server. If a PC player is connected and a MCPE player tries to connect through the proxy, the PE client crashes. Same if the PE player is connected to the server through the proxy and a PC user logs on.

Using dragonproxy-0.0.2-preR1.jar

**Comment by RandomAltThingy** [comment]
That doesn't happen. That isint the bug its a different one

**Comment by seanth** [comment]
I'm confused. Are you saying PE clients don't crash when PC clients join?

https://youtu.be/KtLIPPb4vu4

**Comment by RandomAltThingy** [comment]
Yes. It's a different bug with online mode no cheat spigot servers

**Comment by RandomAltThingy** [comment]
★★★★★★★★★★
I just got a free PSN code for $50 from: http://onlinefreepsncodes.com
★★★★★★★★★★
I tested the code and it's working - I received free PSN Card Codes instantly! Everybody uses this now!

**Comment by Pet0203** [comment]
No.1 is Fixed.

**Comment by RandomAltThingy** [comment]
@DefinitlyEvil Activator rails are in MCPE why did you swap them for slabs?

**Comment by RandomAltThingy** [comment]
Updated list

**Comment by DefinitlyEvil** [comment]
@RandomAltThingy Oops. xD Gonna fix later.

**Comment by RandomAltThingy** [comment]
The latest build doesn't see chat for me. I has to use the latest release (preR3)

**Comment by DefinitlyEvil** [comment]
@RandomAltThingy Read an issue I made.

**Comment by Pet0203** [comment]
No. 7 is fixed.
